src/Stratis.Bitcoin.Features.Miner/PosMinting.cs

```diff
@@ -30,10 +30,12 @@ namespace Stratis.Bitcoin.Features.Miner
     /// <see cref="PosConsensusOptions.StakeMinConfirmations"/> blocks ago are eligible for staking.
     /// <para>
     /// The overall process for "attempting" to mine a PoS block looks like this:
-    /// 1) Create new block with transactions from mempool.
-    /// 2) Get UTXOs that can participate in staking (have suitable depth).
-    /// 3) Split these UTXOs in subsets and create tasks processing each subset to allow for parallel processing.
-    /// 4) Each of the tasks mentioned above will try to find a solution for proof of stake target. This is done by creating a coinstake
+    /// <list>
```

Comment:
almost :) you do have 5) out of list and you should not need prefixes like 1), 2) in list if you use `<list type="number">`